kolibri/core/discovery/api.py

```diff
@@ -77,6 +77,7 @@ class StaticNetworkLocationViewSet(NetworkLocationViewSet):
 
 class NetworkLocationFacilitiesView(BaseValuesViewset):
     queryset = NetworkLocation.objects.all()
+    values = ()
```

Comment:
This is just a quick fix I found to be able to list the devices. Should be removed before merge.